### DISCUSSION TITLE
i18n Postal

### DISCUSSION THREAD

**Discussion by luizeof** [postalserver/postal#1470] [Feature suggestions]
Hello Folks from Postal ... This App is Awesome!

Can we make an effort to translate Postal? Do you have some guidelines to do this with your App?

Our team can translate Postal, but we want to know your plans ....

Very Very Thanks!!!

**Comment by mikecmpbll** [comment]
not a part of the team, but I would guess it's a case of replacing all english language fragments with translations in https://github.com/atech/postal/blob/master/config/locales/en.yml , and then writing an additional translation file for each new language.

**Comment by luizeof** [comment]
@mikecmpbll  @adamcooke Hello Guys make i18n will take a Lot of time ... can I make an pull for each module or one pull after all ?

**Comment by willpower232** [comment]
One pull request to move the English language fragments to that locale file and then each additional language as its own pull request is probably enough separation in this case.

The initial moving of the language fragments would be the biggest one and it looks like there are a few unnecessary terms kept in the current en locale file.
